perun-core/src/main/java/cz/metacentrum/perun/core/bl/ConsentsManagerBl.java

```diff
@@ -98,4 +101,25 @@
 	 * @throws ConsentHubExistsException if consent hub with the same name exists
 	 */
 	ConsentHub updateConsentHub(PerunSession perunSession, ConsentHub consentHub) throws ConsentHubExistsException;
+	/**
+	 * Adds facility to consent hub.
+	 *
+	 * @param sess session
+	 * @param consentHub consent hub
+	 * @param facility facility to be added
+	 * @throws FacilityAlreadyAssigned if facility is already assigned to consent hub
+	 */
+	void addFacility(PerunSession sess, ConsentHub consentHub, Facility facility) throws FacilityAlreadyAssigned;
+
+	/**
+	 * Removes facility from consent hub.
```

Comment:
Please, mention here that the consent hub is deleted if it is the last facility in the consent hub because it is a little bit unexpected behavior.